bin/hex_cone.json

```diff
@@ -5,5 +5,8 @@
 	"material_properties" : {
 		"density" : 2000.0,
 		"k0" : 0.5
-	}
+	},
+	"element_type" : 5,
+	"dimension" : 3,
+	"vertices" : 8
```

Comment:
Shouldn't element type define vertices? or am I missing something?

Reply:
Nope, we can have higher order elements too. For instance, 4-node quadrilateral is a `3`, 9-node quadrilateral is `10` and 8-node quadrilateral is `16`.

More info is here: http://gmsh.info/doc/texinfo/gmsh.html#File-formats

Reply:
Yes, but why should I have to specify that in the input JSON file, can we not deduct the type of element from the mesh file?

Reply:
Understood your concern now. Hmm, my concern is we have templatized `Tvertices` and we need them. Should we do something in `io` to store `nvertices` given certain `element_type`? I feel that is quite a hassle too

Reply:
`IO` class is `IO` it shouldn't worry about `nvertices`. Use a factory to create the right `element_type`